src/main/java/org/terracotta/statistics/StatisticsManager.java

```diff
@@ -92,26 +109,38 @@ private static void parseStatisticAnnotations(final Object object) {
         } else if (Modifier.isStatic(m.getModifiers())) {
           throw new IllegalArgumentException("Statistic methods must be non-static: " + m);
         } else {
-          StatisticsManager.createPassThroughStatistic(object, anno.name(), new HashSet<>(Arrays.asList(anno.tags())), new MethodCallable<>(object, m));
+          StatisticsManager.createPassThroughStatistic(object, anno.name(), new HashSet<>(Arrays.asList(anno.tags())), supply(anno.type(), new ReflectionSupplier<>(object, m)));
         }
-      } 
+      }
     }
   }
 
-  static class MethodCallable<T> implements Callable<T> {
+  public static Map<String, String> properties(String... kvs) {
+    return Stream.of(kvs)
+        .map(kv -> kv.split("="))
```

Comment:
I'm pretty sure you want a limited split here: `.map(kv -> kv.split("=", 2))` and you should probably `Pattern.quote("=")` for sanity.